docs/src/development/proposals/MEP12/README.md

```diff
@@ -0,0 +1,43 @@
+# Rack Spreading
+
+Currently, when creating a machine through the metal-api, the machine is placed randomly inside a partition. This algorithm does not consider spreading machines across different racks and different chassis. This may lead to the situation that a group of machines (that for example form a cluster) can end up being placed in the same rack and the same chassis.
+
+Spreading a group of machines across racks can enhance availability for scenarios like a rack loosing power or a chassis meltdown.
+
+So, instead of just randomly deciding the placement of a machine candidate, we want to propose the introduction of a placement strategy for the machine allocation.
+
+## Placement Strategies
+
+The following placement strategies should be implemented:
+
+- `project`: Machines in the project are spread across all available racks evenly (best effort), the user can optionally pass tags which will be considered for spreading the machines as well (this will for example allow spreading by a cluster id tag inside the same project)
+- `random`: Machines are randomly placed inside the racks (current implementation)
```

Comment:
- Why do you want to keep `"random"` if the default should be `"project"` (best effort) anyway?
- Why would a customer want to choose `"random"`?

Reply:
You are right. Probably the algorithm for `random` is simpler and faster, but I cannot come up with a direct advantage why a user should choose `random`. If nobody else finds an advantage of keeping the old strategy, I think we can remove it entirely.

Reply:
i think if a user needs new machines they will use the default settings no matter if it takes longer or not.

Reply:
Yes lets get rid of the `random` strategy, there is no point keeping it, except we can see that machine allocation will take twice as long as before, which is very unlikely the case.